src/app/components/elements/panels/SelectedEventDetails.tsx

```diff
@@ -103,14 +136,43 @@ export const SelectedEventDetails = ({ eventId }: { eventId: string }) => {
             <br />
             <strong>Number of teachers: </strong>
             {teacherCount} / {selectedEvent.numberOfPlaces}
-          </p>
+            <br />
+            <strong>Gender:</strong>
+            <span id={`gender-stats-tooltip`} className="icon-help ml-1" />
+            <UncontrolledTooltip className="text-nowrap" target={`gender-stats-tooltip`} placement="right">
+              User gender of CONFIRMED or ATTENDED bookings
+            </UncontrolledTooltip>
+            <br />
+            <ListGroup>
+              <ListGroupItem className="py-0">{`Male: ${male} (${asPercentage(
```

Comment:
Copy/pasted `ListGroupItem`. This could be easily refactored. For instance, you could have a component that can live in this file that accepts a `Gender`, a count, and a numberOfConfirmedOrAttendedBookings.

Reply:
In fact the ListGroup itself could be its own component too. You could pass the list of event bookings and it would be in there where you would calculate the numberOfConfirmedOrAttendedBookings and where you would count the genders. That way it's all in the right place, rather than having `SelectedEventDetails` do 20 million things.

Reply:
Moved this (though kept in the same file as it's not used elsewhere) and added a couple of tests